src/librustc_typeck/check/method/confirm.rs

```diff
@@ -566,10 +566,7 @@ impl<'a, 'tcx> ConfirmContext<'a, 'tcx> {
         &self,
         predicates: &ty::InstantiatedPredicates<'tcx>,
     ) -> Option<Span> {
-        let sized_def_id = match self.tcx.lang_items().sized_trait() {
-            Some(def_id) => def_id,
-            None => return None,
-        };
+        let sized_def_id = self.tcx.lang_items().sized_trait().require().ok()?;
```

Comment:
Should we error in places like this when we don't have the `Sized` trait? I'm not sure if we really need to gracefully handle its absence.